video/out/wldmabuf/context_wldmabuf.c

```diff
@@ -37,7 +37,7 @@ static bool init(struct ra_ctx *ctx)
 const struct ra_ctx_fns ra_ctx_wldmabuf = {
     .type               = "none",
     .name               = "wldmabuf",
-    .hidden             = true,
+    .description        = "Wayland DMA-BUF",
```

Comment:
```suggestion
    .description        = "Wayland/DMA-BUF",
```
To make it similar with others?